src/translations/cs.json

```diff
@@ -0,0 +1,41 @@
+{
+  "log_in": "Přihlásit se",
+  "log_out": "Odhlásit se",
+  "loged_in_as": "Přihlášen jako",
+  "logged_in": "Přihlášen",
+  "logging_in": "Přihlášení",
+  "logging_out": "Odhlášení",
+  "sign_up": "Zaregistrovat se",
+  "signing_up": "Registrace",
+  "remember_password": "Zapomněli jste heslo?",
+  "recover_password": "Obnovit heslo",
+  "send_recovery_email": "Odeslat e-mail pro obnovení",
+  "sending_recovery_email": "Odesílání e-mailu pro obnovení",
+  "never_mind": "Nevadí",
```

Comment:
Similarly to the Slovak translation suggestion, what do you think of using something like 'Zpět'?